src/GraphQLParser/Source.cs

```diff
@@ -9,18 +9,11 @@ public Source(string body) : this(body, "GraphQL")
         public Source(string body, string name)
         {
             Name = name;
-            Body = MonetizeLineBreaks(body);
+            Body = body ?? string.Empty;
         }
 
         public string Body { get; set; }
 
         public string Name { get; set; }
-
-        private static string MonetizeLineBreaks(string input)
```

Comment:
Once I already wanted to remove this method but did not dare.

Reply:
Yeah I debated allowing the behavior to be controlled using an optional constructor parameter but reasoned that I was overcomplicating things. I think the only time someone could potentially encounter an issue with this change is if the query contains a string parameter and an implementation is relying on the `\r\n` -> `\n` conversion the parser is performing. That seemed unlikely (but not impossible).